lib/Jsrt/JsrtCommonExports.inc

```diff
@@ -5,6 +5,7 @@
     JsAddRef
     JsRelease
     JsCreateContext
+    JsGetContextUrl
```

Comment:
Alright, I'll see what I can do. I just don't know what kind of information I can get in ch regarding the caller of a WScript method.
Since there can be multiple file after various `WScript.LoadScriptFile()`. Maybe I can make it that the methods returns the path of the initial script (ie from argv) and that might be good enough for the time being.